app/assets/stylesheets/custom.scss

```diff
@@ -161,6 +161,28 @@ th {
   text-align: center;
 }
 
+// user order page
+.order-userinfo {
+  background-color: #e3e3e3;
+  border: 2px solid #6699ff;
+  border-radius: 20px;
+  margin-bottom: 2px;
```

Comment:
Properties should be ordered background-color, border, border-radius, margin, margin-bottom, min-height, padding, width